### PR TITLE
JBR-7028 Implement FPS counter on Linux

### PR DESCRIPTION
**Pull request by mkartashev**
[JBR-7028](https://youtrack.jetbrains.com/issue/JBR-7028) Implement FPS counter on Linux

Please review this from the point of view of usability both on the provider and the supplier side. We will add JBRApi for external access in a separate commit.

I suggest that those toolkits that can provide more or less reliable hardware FPS use the same counter name; here, it is `java2d.native.frames`, but any other will do as well.

### CODE REVIEW

**Review by avu — APPROVED** [review]
LGTM

File: src/java.desktop/share/classes/java/awt/Window.java [hunk 1]
```diff
@@ -4503,13 +4506,99 @@ public void setTrayIconWindow(Window w, boolean isTrayIconWindow) {
...
+            public void bumpCounter(Window w, String counterName) {
+                Objects.requireNonNull(w);
+                Objects.requireNonNull(counterName);
+
+                PerfCounter newCounter;
+                long curTimeNanos = System.nanoTime();
+                synchronized (w.perfCounters) {
+                    newCounter = w.perfCounters.compute(counterName, (k, v) ->
+                            v == null
+                            ? new PerfCounter(curTimeNanos, 1L)
```
Comment (bourgesl) [inline src/java.desktop/share/classes/java/awt/Window.java:4519]: Where is the PerfCounter class ? Missing import ?

Comment (mkartashev) [reply src/java.desktop/share/classes/java/awt/Window.java:4519]: It's further down at line 4597. Simply a two-field record
```
private record PerfCounter(Long updateTimeNanos, Long value) {}
```

**Review by dmitriimorskii — APPROVED** [review]

**Review by bourgesl — APPROVED** [review]

**Comment by mkartashev** [comment]
Merged into `jbr21` as 9d011d23dfd61f82892a98a072eb88016debcf50
A separate commit will be prepared for `main` as it lacks the Wayland toolkit.

**Pull request closed without merging**